Лабораторная работа 1/task_2.py

```diff
@@ -0,0 +1,7 @@
+book = 4 * 25 * 50 * 100
+disk = 1.44 * 1024 * 1024
+count = disk / book
```

Comment:
```suggestion
count = disk // book
```
Использование целочисленного деления более верно, так как последующее округление может привести к округлению в большую сторону, что будет противоречить действительности